client/templates/app/notifications/notificationsButton.js

```diff
@@ -1,6 +1,6 @@
 // See notificationsHelpers.js for helpers
```

Comment:
Since we're different levels of experience, felt like it could be helpful to see where it was connected.. but can remove, np